composeApp/src/main/kotlin/io/github/couchtracker/ui/screens/viewing/CreateViewingDialog.kt

```diff
@@ -0,0 +1,126 @@
+import androidx.compose.foundation.lazy.LazyListScope
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.FilterChip
+import androidx.compose.material3.FilterChipDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.ModalBottomSheet
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.material3.rememberModalBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import io.github.couchtracker.R
+import io.github.couchtracker.utils.Text
+import io.github.couchtracker.utils.str
+import io.github.couchtracker.utils.toText
+import kotlinx.coroutines.launch
+import kotlin.time.Duration
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun CreateViewingDialog(
+    showDialog: MutableState<Boolean>,
+    approximateVideoRuntime: Duration,
+) {
+    val scope = rememberCoroutineScope()
+    val sheetState = rememberModalBottomSheetState()
+    val dateTimeSectionState = rememberDateTimeSectionState()
+
+    if (showDialog.value) {
+        DateTimeSectionDialog(dateTimeSectionState)
+        ModalBottomSheet(
+            onDismissRequest = {
+                showDialog.value = false
+            },
+            sheetState = sheetState,
+        ) {
+            LazyColumn {
+                item {
+                    Text(
+                        R.string.mark_as_watched.str(),
+                        modifier = Modifier.padding(horizontal = 16.dp),
+                        style = MaterialTheme.typography.titleLarge,
+                    )
+                    Spacer(Modifier.height(16.dp))
+                }
+                dateTimeSection(dateTimeSectionState, approximateVideoRuntime)
+                fakeSection("Resolution", listOf("4K", "1080p", "720p"))
+                fakeSection("Place", listOf("Plane", "Car", "Camionetta"))
+                item {
+                    TextButton(
+                        onClick = {
+                            scope.launch { sheetState.hide() }.invokeOnCompletion {
+                                if (!sheetState.isVisible) {
+                                    showDialog.value = false
+                                }
+                            }
+                        },
+                    ) {
+                        Text("Hide bottom sheet")
+                    }
+                }
+            }
+        }
+    }
+}
+
+fun LazyListScope.section(title: Text, content: @Composable () -> Unit) {
```

Comment:
This function is public and just called `section`. Either make it private or name it in a more specific way, e.g. `watchedItemDialogSection`.